layouts/partials/alert.html

```diff
@@ -3,7 +3,7 @@
         <a href="{{ relref . "/topics/kubernetes.md" }}" class="block p-2 bg-purple-400 hover:bg-purple-300 transition-all items-center text-purple-100 leading-none rounded-full flex md:inline-flex">
             <span class="flex rounded-full bg-orange-700 border-4 border-orange-600 uppercase text-white px-2 py-1 text-xs font-bold mr-2">New</span>
             <span class="text-left flex-auto text-white text-sm md:text-base leading-normal">
-                Check out Pulumi’s new Kubernetes superpowers.
+                Check out how we're making Kubernetes more awesome.
```

Comment:
None of the candidates in https://github.com/pulumi/marketing/issues/11 worked as a link /kubernetes, so I went with a slight variation on @zchase's suggestion. I'll leave it to @isaac-pulumi to either bless this as an interim solution or suggest something else here before we commit.